lti_consumer/lti_xblock.py

```diff
@@ -249,13 +251,13 @@ class LtiConsumerXBlock(StudioEditableXBlockMixin, XBlock):
         scope=Scope.settings
     )
 
-    # LTI 1.3 fields
     lti_version = String(
         display_name=_("LTI Version"),
         scope=Scope.settings,
         values=[
             {"display_name": "LTI 1.1/1.2", "value": "lti_1p1"},
             {"display_name": "LTI 1.3", "value": "lti_1p3"},
+            {"display_name": "External Configuration", "value": "external"},
```

Comment:
@tecoholic Which of the options below do you think is better?
- This solution, with the `external` value here
- A separate `List` variable, where the options are either `New configuration` or `Use external configuration`